projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -18,7 +18,49 @@ import React from 'react';
  */
 import styles from './styles.module.scss';
 
+const getConnectionsDiv = () => {
+	// eslint-disable-next-line no-undef
```

Comment:
If you take the namespaced approach I mentioned above, you can add `jetpackSocial` to the `globals` option in `.eslintrc` to avoid having to ignore this rule every time.